web/server/entrypoint.go

```diff
@@ -427,3 +436,48 @@ func (s *robotServer) serveWeb(ctx context.Context, cfg *config.Config) (err err
 	}
 	return web.RunWeb(ctx, myRobot, options, s.logger)
 }
+
+// dumpResourceRegistrations prints all builtin resource registrations as a json array
+// to stdout. If you edit this function, ensure that etc/system_manifest/main.go is
+// updated correspondingly.
+func dumpResourceRegistrations() error {
+	type resourceRegistration struct {
+		API             string             `json:"api"`
+		Model           string             `json:"model"`
+		AttributeSchema *jsonschema.Schema `json:"attribute_schema,omitempty"`
+	}
```

Comment:
I've left this as a private function-only definition to lower the risk that someone else sees this and modifies / uses it.

This means that the test also needs a copy of this definition which is a bit repetitive.